### PR TITLE
Fixes up salvaging a bit, adds new blueprints

### PR DESCRIPTION
**Pull request by AdamElTablawy**
We had a deficit of blueprints and crafting sucks.

While not making crafting any less shit, I sped up salvaging and removed some useless salvage in favor of useful rewards, and I increased the available amount of blueprints and the lootpools that spawn them.

You can now have blueprints for the ppsh-41, the Delisle, the Neostead, the gauss rifle, the m1 carbine, the r82, the uzi, the trail carbine, the .556 pistol, the 10mm smg, the scout carbine, the lever action, the AEP7, the service rifle...

Basically, you have a lot more lootable craftable blueprints.

The speed at which you salvage has also been decreased

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

:cl:
add: More gun blueprints have been added.
tweak: Salvaging is less AIDS.
/:cl:

### CODE REVIEW

File: code/game/objects/items/granters.dm [hunk 1]
```diff
@@ -766,6 +766,21 @@
 	icon_state = "blueprint2"
 	crafting_recipe_types = list(/datum/crafting_recipe/gun/lsw)
 
+/obj/item/book/granter/crafting_recipe/blueprint/m1carbine
+	name = "m1 carbine blueprint"
+	icon_state = "blueprint2"
+	crafting_recipe_types = list(/datum/crafting_recipe/gun/lsw)
```
Comment (buffyuwu) [inline code/game/objects/items/granters.dm:772]: did you mean to put /datum/crafting_recipe/gun/lsw here?

File: code/game/objects/structures/wrecks.dm [hunk 2]
```diff
@@ -355,8 +355,8 @@
 		if(!I.use_tool(src, user, 0, volume=100)) //here is the dilemma, use_tool doesn't work like do_after, so moving away screws it(?)
 			inuse = FALSE
 			return //you can't use the tool, so stop
-		for(var/i1 in 1 to 2) //so, I hate waiting 30 seconds straight... what if we wait 10 seconds 3 times? (yes, its the same, but it'll feel more!)
-			if(!do_after(user, 10 SECONDS*W.toolspeed, target = src)) //this is my work around, because do_After does have a move away
+		for(var/i1 in 1 to 2) //so, I hate waiting
```
Comment (buffyuwu) [inline code/game/objects/structures/wrecks.dm:358]: lol

File: code/modules/crafting/items.dm [hunk 3]
```diff
@@ -937,60 +937,45 @@
...
 				/obj/item/stock_parts/cell/ammo/ec,
-				/obj/item/stack/crafting/goodparts
+				/obj/item/stack/crafting/goodparts/five
 				)
 
 /obj/item/salvage/crafting
 	name = "salvaged components"
 	desc = "Some salvaged components, it could contain some useful materials if dissasembled using a workbench..."
 	icon_state = "salvagecomponents"
-	Loot = list(/obj/item/crafting/diode,
```
Comment (buffyuwu) [inline code/modules/crafting/items.dm:956]: why're we deleting a lot from this list here?

**Review by buffyuwu — CHANGES_REQUESTED** [review]

File: code/modules/crafting/items.dm (hunk 3 above)
Comment (AdamElTablawy) [reply code/modules/crafting/items.dm:956]: because basically, most of these crafting components are no longer used in game, so they're all trash

we can re-add them later, but these are basically unused ever since tinkering ceased to exist

**Review by buffyuwu — COMMENTED** [review]
should be ok to test

**Review by buffyuwu — APPROVED** [review]
